client/src/app/shared/instance/instance-features-table.component.ts

```diff
@@ -50,8 +50,10 @@ export class InstanceFeaturesTableComponent implements OnInit {
         value: config.signup.allowed
       },
       {
-        label: this.i18n('Video uploads require manual validation by moderators'),
-        value: config.autoBlacklist.videos.ofUsers.enabled
+        label: config.autoBlacklist.videos.ofUsers.enabled ?
+          this.i18n('Video uploads require manual validation by moderators') :
+          this.i18n('Uploads are published without moderator verification'),
+        value: true
```

Comment:
please use the variable instead

Reply:
? the whole goal of this pr is to not do any judgment about how you manage your instances, so `value:` must all ways stay at `true` and the ternary operator switch the message by a message or its opposite (and if I invert the sense of the message I must also invert the value wich finish by always false are always true, I choosed always true because that more friendly).
I'm not realy sure about what you are talking, so wich variable do you talk ?

Reply:
Could we imagine something like 

```
label: 'Video upload'
value: 'Requires manual validation by moderators' or 'Automatically published'
```

?

Reply:
@Chocobozzz I've tried but, I find that quite uggly :
![Capture](https://user-images.githubusercontent.com/24391983/59553313-1e1db400-8f8b-11e9-9b35-8141cf4cd676.PNG)
The space in the bottom of Video upload is way too big and the 4 wrap line for a small sentance isn't very pretty.
So I've tried to get a use of this big open blank space in the middle of the screen but that create some problems with reponsiveness.

Since my view is very subjective, are you sure you want this ?

Reply:
Yes I think this is the clearest to make things understood. Just try to increase the `min-width` value in https://github.com/Chocobozzz/PeerTube/blob/develop/client/src/app/shared/instance/instance-features-table.component.scss#L10